deployment/main.tf

```diff
@@ -329,4 +338,32 @@ resource "google_kms_key_ring" "firmware_release_prod" {
 #   }
 # }
 
+resource "google_kms_key_ring" "terraform_state" {
+  name = "armored-witness-bucket-tfstate"
+  location = "europe-west2"
+}
+
+resource "google_kms_crypto_key" "terraform_state_bucket" {
+  name            = "terraform-state-bucket"
+  key_ring        = google_kms_key_ring.terraform_state.id
+  rotation_period = "86400s"
+
+  lifecycle {
```

Comment:
Just wanted to ask whether this was intended to stay in here, or if it's just during dev?

Reply:
I got it from the doc, but looks like it's unnecessary. Removed.